tests/clients/test_ngsi_v2_cb.py

```diff
@@ -961,6 +996,22 @@ def test_batch_operations(self):
             self.assertEqual(1000,
                              len(client.query(query=query,
                                               response_format='keyValues')))
+            # update with keyValues
+            entities_keyvalues = [ContextEntityKeyValues(id=str(i),
+                                                         type=f'filip:object:TypeC',
+                                                         attr1="text attribute",
+                                                         attr2=1
+                                                         ) for i in range(0, 1000)]
+            client.update(entities=entities_keyvalues,
+                          update_format="keyValues",
+                          action_type=ActionType.APPEND)
```

Comment:
@djs0109 Is this intended by FIWARE or why can you create multiple entities with the same ID?

Reply:
I think the IDs are different, from 0 to 1000. Or does it show something else in your test?

Reply:
@djs0109 You can only see that if you look at each specific type separately in the GET request for the OCB endpoint, for example:

`GET http://localhost:1026/v2/entities?limit=1000&type=filip:object:TypeA`

and 

`GET http://localhost:1026/v2/entities?limit=1000&type=filip:object:TypeC`

gives me entities with different types but with the same ID

Reply:
You can also use GET http://localhost:1026/v2/entities?limit=1000&id=0

Reply:
@RCX112 ok, I understand what you mean now. Yes, the IDs of different types are the same. It is fine here, but still thanks for pointing it out